.github/workflows/test-windows.yml

```diff
@@ -45,9 +41,17 @@ jobs:
         uses: actions/setup-python@61a6322f88396a6271a6ee3565807d608ecaddd1  # v4.7.0
         with:
           python-version: ${{ env.PYTHON_VERSION }}
-      - name: Install deps
-        run: choco install nasm
       - name: Environment Information
         run: npx envinfo
+      - name: Install deps
+        run: choco install nasm
       - name: Build
         run: ./vcbuild.bat
+      - name: Test CI JS
+        run: ./vcbuild.bat release noprojgen nobuild ignore-flaky test-ci-js
+        env:
+          test_ci_args: --mode=release --flaky-tests=dontcare --run=0,4 --measure-flakiness 9 -p tap --logfile test.tap default pummel
```

Comment:
`--run=0,4` is only running a quarter of the tests.

Reply:
Why change to tap reporter?

Reply:
From what I saw, the tests on Jenkins is configured to run just a quarter too.

About tap reporter, I just because it worked but I can try replace for actions and see if the output is better.

Reply:
> From what I saw, the tests on Jenkins is configured to run just a quarter too.

On Jenkins the tests are split up and run in separate jobs (each job running a quarter). Altogether all the tests are run. 
e.g. All of these jobs make up one run of `test-ci-js` (note that `RUN_SUBSET` changes, which ends up as `--run=0,4`, `--run=1,4`, `--run=2,4`, etc.):
- https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2016-COMPILED_BY-vs2022-x86/24289/console
- https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win2016-COMPILED_BY-vs2022-x86/24289/console
- https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2016-COMPILED_BY-vs2022-x86/24289/console
- https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2016-COMPILED_BY-vs2022-x86/24289/console
![image](https://github.com/nodejs/node/assets/5445507/e44e1ca4-8b6b-45bd-8a6e-3fa688bd9921)

Reply:
Oh, okay, so in this case, do you want to run all the tests or just run one quarter?

Reply:
All of them.